.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "<app name>: <description>"
+labels: kind/bug, triage/support
+assignees: ''
+
+---
+
+#### Describe the bug
+
+A clear and concise description of what the bug is.
+
+#### Environment info
+
+#### Files
+
+A list of relevant files for this issue. This will help people navigate the project and offer some clues of where to start.
+
+#### Steps to reproduce
+
+Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '...'
+3. Scroll down to '...'
+4. See error
+
+#### Expected behaviour
+
+A clear and concise description of what you expected to happen.
+
+#### Screenshots
+
+If applicable, add screenshots to help explain your problem.
+
+#### Additional context
+
+Add any other context about the problem here.
+
+Remember to use helpful labels. If you use the "help wanted" label, it will be easier for contributors and maintainers of the project to identify your ticket.
```

Comment:
Possibly list a few labels that could be used.

Maybe this should be written in "comment" format aka won't show once the issue is created